elements/rh-navigation-vertical/rh-navigation-vertical-group.ts

```diff
@@ -0,0 +1,167 @@
+import { LitElement, html, isServer, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { state } from 'lit/decorators/state.js';
+import { query } from 'lit/decorators/query.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import { InternalsController } from '@patternfly/pfe-core/controllers/internals-controller.js';
+
+import { consume, provide } from '@lit/context';
+import { context, type RhNavigationVerticalContext } from './context.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
+
+import styles from './rh-navigation-vertical-group.css';
+
+/**
+ * A disclosure menu of grouped navigation items in a vertical navigation list.
+ * @summary Vertical navigation group
+ * @slot - Default slot for navigation items or additional sub groups.
+ * @slot summary - A summary slot for the group title, overrides the summary attribute.
+ */
+@customElement('rh-navigation-vertical-group')
```

Comment:
this is basically rh-disclosure, except we have to maintain and support them separately. see previous rant re tag name creep.

imagine a world in which we lazily register parent-specific behaviours on a shared controller in rh-disclosure, such that we get to reuse it where we want, without burdening all pages with all the code.

Reply:
We could use rh-disclosure here, but we would need to revisit how we allow styling of it.